test/ibmq/test_basic_server_paths.py

```diff
@@ -120,3 +149,14 @@ def _submit_job_with_retry(self, circs, backend, max_retry=5):
                 time.sleep(1)
 
         self.fail("Unable to submit job after {} retries: {}".format(max_retry, limit_error))
+
+    def _cancel_job(self, job: RuntimeJob) -> None:
```

Comment:
cancel_job is already imported from utils in this file. Could you use that and remove this?

Reply:
Updated! all definitions of `_cancel_job` were removed in favor of `cancel_job` from utils.